### PR TITLE
Fix bundle exec calls

### PR DESCRIPTION
**Pull request by jquick**
With Inspec 2.0 we have removed parts of the CLI. This PR updates supermarket and compliance to be fully functional again.

Fixes https://github.com/chef/inspec/issues/2668

Signed-off-by: Jared Quick <jquick@chef.io>

### CODE REVIEW

**Review by miah — APPROVED** [review]

File: test/functional/inspec_supermakert_test.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  it 'info' do
+    out = inspec('supermarket info dev-sec/ssh-baseline')
+    out.exit_status.must_equal 0
+    out.stderr.must_equal ''
+    out.stdout.must_include "name: \e[0m  ssh-baseline"
+  end
+
+  it 'supermarket exec' do
+    out = inspec('supermarket exec dev-sec/ssh-baseline')
```
Comment (jerryaldrichiii) [inline test/functional/inspec_supermakert_test.rb:22]: Can we add a test for exit status here as well? Even if it is `wont_equal 1`.

**Review by jerryaldrichiii — CHANGES_REQUESTED** [review]
Looks great! Only one small ask.

File: test/functional/inspec_supermakert_test.rb (hunk 1 above)
Comment (jquick) [reply test/functional/inspec_supermakert_test.rb:22]: yeah we can do wont_equal 1, as it could be 0, 100, 101

**Review by jerryaldrichiii — APPROVED** [review]
Looks great! Thanks @jquick

**Review by chris-rock — APPROVED** [review]
Thank you @jquick

**Pull request merged**